.github/workflows/container-healthchecks.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           # Ignore mocks since they don't have health checks
           # TODO: add health check for svc-bgs-api; ignore it for now
           if docker container ls --format '{{.Names}} \t {{.Status}}' | grep -v "(healthy)" \
-            | grep -v "svc-bgs-api\|-mock-"; then
+            | grep -v "svc-bgs-api\|mock-"; then
```

Comment:
great catch in identifying why this filtered list of containers is failing to exclude all mock containers.